Remove "setup" page, make the landing page a quickstart with code (#144)
langchain-ai/langsmith-docs: docs/index.mdx

  LangChainQuickStartCodeTabs,
  ConfigureEnvironmentCodeTabs,
  RunTreeQuickStartCodeTabs,
  ConfigureSDKEnvironmentCodeTabs,
  PythonSDKTracingCode,
  TypeScriptSDKTracingCode,
} from "@site/src/components/QuickStart";
import { ClientInstallationCodeTabs } from "@site/src/components/ClientInstallation";
import DocCardList from "@theme/DocCardList";

# Getting started with LangSmith

## Introduction

> **LangSmith** is a platform for building production-grade LLM applications. Monitor and evaluate your application in real-time, allowing you to ship quickly and with confidence. Use of LangChain is not necessary - LangSmith works on its own!

Author: Added explicit statement that LangSmith works without LangChain. However, you do need to import langchain to use runOnDataset in Typescript. Think it's still fine to call out here? I will add a linear ticket to remove any reference to LangChain from this page once we can evaluate in the typescript SDK directly. cc @hinthornw

> **LangSmith** is a platform for building production-grade LLM applications. Monitor and evaluate your application in real-time, allowing you to ship quickly and with confidence. Use of LangChain is not necessary - LangSmith works on its own!

Reviewer: ```suggestion
**LangSmith** is a platform for building production-grade LLM applications. It allows you to closely monitor and evaluate your application, allowing you to ship quickly and with confidence. Use of LangChain is not necessary - LangSmith works on its own!
```
